explainer-client/src/main/scala/ExplainEditorJSDomBuilders.scala

```diff
@@ -23,10 +23,20 @@ import scala.scalajs.js.JSON
 object ExplainEditorJSDomBuilders {
 
   def statusBarText(explainer: CsAtom) = {
-    val isPublished = explainer.contentChangeDetails.published.isDefined
 
-    if(isPublished) {
-      "published"
+    def getLongFromOptionCSChangeRecordForComparison(optionCSChangeRecord : scala.Option[CsChangeRecord]) : Long = {
+      optionCSChangeRecord match {
+        case Some(cr) => cr.date
+        case None => 0L
+      }
+    }
+    
+    if(explainer.contentChangeDetails.published.isDefined) {
+      if ( getLongFromOptionCSChangeRecordForComparison(explainer.contentChangeDetails.published) < getLongFromOptionCSChangeRecordForComparison(explainer.contentChangeDetails.lastModified) ){

```

Comment:
I think something like this would be cleaner and avoid the need for a second function:

    val changesSincePublication = for {
      p <- explainer.contentChangeDetails.published
      lm <- explainer.contentChangeDetails.lastModified
    } yield {
      if (p.date < lm.date) "unseen" else "seen"
    }
      changesSincePublication.getOrElse("draft")